.changeset/green-bags-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes': minor
```

Comment:
I believe this should be a patch since we are below version 1 and it should be breaking(?)

Reply:
Yup, patch is the new minor in sub 1 version.

*not breaking, I assume. Looks to be additional

Reply:
Done!